sys/kern/kern_clock.c

```diff
@@ -531,7 +534,10 @@ hardclock(int cnt, int usermode)
 			if (i > 0 && i <= newticks)
 				watchdog_fire();
 		}
-		intr_event_handle(clk_intr_event, NULL);
+#ifdef DEV_ACPI
+		if (!CK_SLIST_EMPTY(&clk_intr_event->ie_handlers))
+			intr_event_handle(clk_intr_event, NULL);
```

Comment:
@jrtc27 was objecting to this on #1358, due to looking inside `struct intr_event`.  Issue would be, what approach should instead be used?  Perhaps clock interrupts aren't considered stray if there are no handlers?  Perhaps a flag to indicate no handler shouldn't be considered stray?